### ISSUE TITLE
[BUG] Cannot Login or Run Tests on M1 Processor

### ISSUE THREAD

**Issue by ismisepaul** [OWASP/SecurityShepherd#626]
**Describe the bug**
Cannot build or test on Mac M1 because of Argon2 - more https://github.com/java-native-access/jna/issues/1313

**To Reproduce**
Try run tests or login to the application once its up and running

**Expected behavior**
Able to run tests and login

 - OS: macOS Monterey 12.3.1
 - Browser Chrome 100

**Further Information:**
Error message

```
15-Apr-2022 21:19:49.824 SEVERE [http-nio-8080-exec-5] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [/login] in context with path [/owaspSecurityShepherd] threw exception [Servlet execution threw an exception] with root cause
	java.lang.NoClassDefFoundError: Could not initialize class de.mkammerer.argon2.Argon2Factory$Argon2Types
		at de.mkammerer.argon2.Argon2Factory.create(Argon2Factory.java:21)
		at dbProcs.Getter.authUser(Getter.java:131)
		at servlets.Login.doPost(Login.java:84)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at servlets.SetupFilter.doFilter(SetupFilter.java:40)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:543)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:698)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:367)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:639)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:882)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1647)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191)
		at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by ismisepaul** [comment]
Pull request #609 solves the problem

**Comment by ismisepaul** [comment]
fixed in #638
